### DISCUSSION TITLE
I'd like to show the connection line after a user has clicked a Handle

### DISCUSSION THREAD

**Discussion by Vochsel** [xyflow/xyflow#3002] [Q&A]
I see for touch devices you can connect nodes on a touch device by tapping two handles in a row.

This behaviour also works on desktop, but I would like to show the connection line to the mouse whilst this is in progress.

I found the onClickConnectStart event, but unsure how to properly show the connection line.

Anyone done this before or have tips?

Thanks!

**Comment by hmaiermack** [comment]
Haven't dealt with this, but the way I'd approach this problem is by creating the edge on click and then updating the edge's target coordinates based on mouse position.

**Reply by moklick** [reply]
Since 11.5 you can also connect nodes on a touch device via drag. This wasn't possible before and the on click connect thing was a workaround.
